apps/docs/pages/guides/storage/cdn.mdx

```diff
@@ -66,6 +66,54 @@ If your asset is updated frequently, we recommend that you re-upload the new ass
 
 Objects in public buckets do not require any Authorization to access objects. This leads to a better cache hit rate compared to private buckets. For private buckets, permissions for accessing each object is checked on a per user level. For example, if two different users access the same object in a private bucket from the same region, it results in a cache miss for both the users since they might have different security policies attached to them. On the other hand, if two different users access the same object in a public bucket from the same region, it results in a cache hit for the second user.
 
+## Debugging Cache Hits
+
+The [storage report](https://supabase.com/dashboard/project/_/reports/storage) will give a breakdown of your project's current storage request caching.
+
+A high cache hit rate is desirable to reduce storage egress costs.
+
+The [Logs Explorer](https://supabase.com/dashboard/project/_/logs/explorer) can also be used to debug cache misses.
```

Comment:
I think its worth adding a note that the data processed by this query will depend on the log retention settings of your plan